### ISSUE TITLE
[Mac OSX Eclipse devel] openHAB_runtime.launch has no UI/Web response, stalls start of the OHAB2 instance

### ISSUE THREAD

**Issue by HentschelT** [openhab/openhab-core#247]
On launching OHAB2 via the devel setup in Eclipse on Mac OSX, the launch may stall, or lead to none of the web UI's responding (web browser spins forever). However, some activity in the log is still observed (add-ons etc). 

The reason is a loading issue for one of the native OSX libraries that comes with the Oracle Java install. Basically, the class loader in the OSGI start thread never returns, but holds a lock on a List or Vector, which in turn blocks other threads. The issue is detailed here: http://bugs.java.com/bugdatabase/view_bug.do?bug_id=8019496

In short: The fix for this is to add '-Djava.awt.headless=true' and '-XstartOnFirstThread' to the VM arguments. 
There is already a check mark in the launch config panel for '-XstartOnFirstThread', perhaps '-Djava.awt.headless=true' can be added there? In the meantime, just add the two options in the VM arguments section.
